### PR TITLE
Fix the animation when removing a toast

### PR DESCRIPTION
**Pull request by lowiebenoot**
### Description

The animation was very weird when you removed a toast notification. This was because indexes were used as keys.

This didn't really occur in the stories, because they were using indexes as keys as well. So I fixed the story as well.

#### Screenshot before this PR

toasts_bugged

#### Screenshot after this PR

toasts_fixed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## [UNRELEASED]
+
+### :sparkles: Added
+
+### :fire: Removed
+
+### :bug: Fixes
+
+- fix the animation when removing a `Toast` from the `ToastContainer`
```
[the remark this answers is not in the document]
Comment (duivvv) [reply CHANGELOG.md]: @lowiebenoot 

something like this?

---
([@lowiebenoot](https://github.com/lowiebenoot) in [#322](https://github.com/teamleadercrm/ui/pull/322))
---

We have the opportunity to make sure this is correct from the start :)

**Comment by duivvv** [comment]
@lowiebenoot would also be nice to add this summary at the bottom cfr. the example on keep a changelog

[Unreleased]: https://github.com/teamleadercrm/ui/compare/v1.0.0...HEAD
[1.0.0]: https://github.com/teamleadercrm/ui/compare/v0.3.0...v1.0.0
[0.3.0]: https://github.com/teamleadercrm/ui/compare/v0.2.0...v0.3.0

**Comment by lowiebenoot** [comment]
I guess it's the same annoying thing here, those links don't exist until you actually release it.

**Comment by nickwaelkens** [comment]
So let's make sure that stuff is in the `CHANGELOG.md` when we make a PR to the bump version. It's all about discipline...

**Comment by nickwaelkens** [comment]
Feel free to merge this now btw 👍

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (duivvv) [reply CHANGELOG.md]: I always create the PR while in progress with a WIP tag, that should fix it no?

**Comment by duivvv** [comment]
@lowiebenoot @nickwaelkens 

You have to change the unreleased to X.X.X at some point, so that makes perfect sense to do it in the 'release' PR.

I would have added the PR reference in then PR itself (see my comment), it's just an extra commit (as if you would make a change after a review), can't see the annoyingness of it.

But too late now.

**Pull request merged**